lib/frontdesk.rb

```diff
@@ -0,0 +1,64 @@
+require "Date"
+
+require_relative "reservation"
+
+module Hotel
+  class FrontDesk
+    attr_accessor :rooms, :reservations, :start_date, :end_date, :room_numbers
+    attr_reader :NUMBER_OF_ROOMS
+
+    def initialize
+      @NUMBER_OF_ROOMS = 20
```

Comment:
You should pick constant or instance variable, but you shouldn't use both for one variable